cli/command/environment_test.go

```diff
@@ -134,8 +133,8 @@ func TestCreateEnvironment(t *testing.T) {
 		req := models.CreateEnvironmentRequest{
 			EnvironmentName:  "env_name",
 			InstanceType:     "t2.small",
-			MinScale:         2,
-			MaxScale:         5,
+			Scale:            2,
+			EnvironmentType:  "static",
```

Comment:
We will need a test to cover the "dynamic" code path.

Reply:
✅